src/main/java/practice/StreamPractice.java

```diff
@@ -48,18 +72,30 @@ public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toA
      * Example: select people of working age
      * (from 18 y.o. and to 60 y.o. for men and to 55 y.o. for women inclusively).
      */
+
     public List<Person> getWorkablePeople(int fromAge, int femaleToAge,
                                           int maleToAge, List<Person> peopleList) {
-        return Collections.emptyList();
+        Predicate<Person> personBasedOnSexAndAge = (person) -> person.getAge() >= fromAge
+                && ((person.getSex() == Person.Sex.MAN && person.getAge() <= maleToAge)
+                || (person.getSex() == Person.Sex.WOMAN && person.getAge() <= femaleToAge));
```

Comment:
```suggestion
        Predicate<Person> isWorkablePeople = (person) -> person.getAge() >= fromAge
                && ((person.getSex() == Person.Sex.MAN && person.getAge() <= maleToAge)
                || (person.getSex() == Person.Sex.WOMAN && person.getAge() <= femaleToAge));
```
it is a good decision  and more readable